config/application.rb

```diff
@@ -53,6 +54,7 @@ class Application < Rails::Application
     config.active_job.queue_adapter = :sidekiq
 
     config.middleware.insert_after ActionDispatch::HostAuthorization, RedirectToServiceGovUkMiddleware
+    config.middleware.use VendorAPITracingMiddleware
```

Comment:
I suppose it doesn't matter where it goes in the stack?

Reply:
It probably does, if we want to keep track of the response time? Not sure though.